hubverse_annotator/utils.py

```diff
@@ -268,68 +268,68 @@ def quantile_forecast_chart(
     """
     if forecast_table.is_empty():
         return alt.layer()
-    value_col = "value"
-    y_axis = alt.Axis(
-        title=f"{selected_target}",
-        grid=grid,
-        ticks=True,
-        labels=True,
-        orient="left",
+    df_wide = forecast_table.filter(pl.col("output_type") == "quantile").pivot(
+        on="output_type_id",
+        index=cs.exclude("output_type_id", "value"),
+        values="value",
     )
-    # filter to quantile only rows and ensure quantiles
-    # are str for pivot; also, pivot to wide, so quantiles
-    # ids are columns
-    df_wide = (
-        forecast_table.filter(pl.col("output_type") == "quantile")
-        .pivot(
-            on="output_type_id",
-            index=cs.exclude("output_type_id", value_col),
-            values=value_col,
-        )
-        .with_columns(pl.col("0.5").alias("median"))
-    )
-    base_x_enc = alt.X(
-        "target_end_date:T",
-        axis=alt.Axis(title="Date", grid=grid, ticks=True, labels=True),
-    )
-    base_y_enc = alt.Y("median:Q", axis=y_axis, scale=alt.Scale(type=scale))
-    base = alt.Chart(df_wide, width=PLOT_WIDTH).encode(
-        x=base_x_enc,
-        y=base_y_enc,
-    )
-    band_95 = base.mark_errorband(
-        extent="ci",
-        opacity=0.1,
-        interpolate="step",
-    ).encode(
-        y=alt.Y("0.025:Q", axis=y_axis),
-        y2="0.975:Q",
-        fill=alt.value("steelblue"),
-    )
-    band_80 = base.mark_errorband(
-        extent="ci",
-        opacity=0.2,
-        interpolate="step",
-    ).encode(
-        y=alt.Y("0.10:Q", axis=y_axis),
-        y2="0.90:Q",
-        fill=alt.value("steelblue"),
+    col_rename_map = {
+        "0.025": "q025",
+        "0.10": "q10",
+        "0.25": "q25",
+        "0.5": "median",
+        "0.75": "q75",
+        "0.90": "q90",
+        "0.975": "q975",
+    }
```

Comment:
write as a function that takes arbitrary quantiles. Can just call `median` `q50`